setup.py

```diff
@@ -8,6 +8,7 @@
     author="Microsoft",
     author_email="",
     description="A pytest wrapper with fixtures for Playwright to automate web browsers",
+    license="MIT License",
```

Comment:
We actually have Apache License, see here: https://github.com/microsoft/playwright-pytest/blob/main/LICENSE